src/pyOpenMS/pxds/AbsoluteQuantitation.pxd

```diff
@@ -29,7 +29,7 @@ cdef extern from "<OpenMS/ANALYSIS/QUANTITATION/AbsoluteQuantitation.h>" namespa
             Param & transformation_model_params,
             Param & optimized_params) nogil except +
 
-        void optimizeCalibrationCurves(
-            libcpp_map[ String, libcpp_vector{ AQS_featureConcentration ]] & components_concentrations) nogil except +
+        # void optimizeCalibrationCurves(
```

Comment:
Did it build after the `{` typo was correct?  I would like to be able to call this function from pyopenms.  Should I just return the changed input instead of changing in place?